dist/10-10EZ-schema.json

```diff
@@ -511,6 +511,15 @@
     "receivesVaPension": {
       "type": "boolean"
     },
+    "vaCompensationType": {
+      "type": "string",
+      "enum": [
+        "none",
```

Comment:
Can we change the order here to:

lowDisability
highDisability
pension
none

It's annoying, but the form library we use doesn't let us re-order enums, so this is the order of the radio buttons you see in the UI.

Reply:
Done! Thanks.